docs/development.rst

```diff
@@ -0,0 +1,56 @@
+Development
+===========
+
+To start with, clone the repo to your local machine ::
+
+    $ git clone git@github.com:caktus/tequila-common.git
+    $ cd tequila-common/
+
+Then, within some existing project, uninstall the pinned version of
+tequila-common and replace it with a symlink to the checkout.  The
+following assumes that you have Ansible configured to install roles
+into the deployment/roles/ directory within your Django project's tree
+::
+
+    $ cd my-django-project/
+    $ workon project
+    (project)$ ansible-galaxy uninstall tequila-common
+    (project)$ cd deployment/roles/
+    (project)$ ln -s /path/to/tequila-common
+
+Now when you make changes to the tasks or files in your checkout of
+tequila-common, those changes will be immediately available when you
+do a deployment.
+
+When finished trying out your changes, be sure to remove the symlink
+and reinstall the pinned version ::
+
+    (project)$ rm deployment/roles/tequila-common
+    (project)$ ansible-galaxy install -r deployment/requirements.yml
+
+
+Documentation
+-------------
+
+In order to build the documentation, you need to have `Sphinx
+<http://www.sphinx-doc.org/en/stable/>`_ and its dependencies
+installed ::
+
+    $ mkvirtualenv tequila -p $(which python3)
```

Comment:
Perhaps we should call this virtualenv tequila-common?

Reply:
I'd like to avoid recommending to people to have different virtualenvs for each of these repos, when they are all going to get the same things installed in them anyway.